### DISCUSSION TITLE
Managing mods

### DISCUSSION THREAD

**Discussion by Tintouin** [Godiesc/firefox-gx#66] [Ideas]
I think there is a way you could manage GX mods, the manifest.json is pretty simple to convert into something understandable for firefox. Since it can have animated backgrounds the sounds and all mods can be considered as extensions.  I think that with some work we can bring the mods to firefox-gx

**Comment by Godiesc** [comment]
hi,
I'm not extension developer so for my side I can't promise to develop "mods" (like opera-gx has) to Firefox. Maybe someone that likes that features could do, that would be great.

**Comment by c3004-a** [comment]
there is an adon that does that but it wont change the background
 https://addons.mozilla.org/de/firefox/addon/gx-mods/
